.gitignore

```
@@ -1,5 +1,7 @@
 .cache
 public/rss.xml
+public/sw.js
+public/workbox-*.js
```

Disse lagres som `sw.js.map` og `workbox-*.js.map` hos meg, kanskje gjøre noe slik?
```suggestion
public/sw.js*
public/workbox-*.js*
```

---

Ja enig